### PR TITLE
feat: add a link to the Anchore UI if the ui URL is set

### PR DESCRIPTION
**Pull request by bradleyjones**
Signed-off-by: Bradley Jones <bradley.jones@anchore.com>

### CODE REVIEW

File: src/main/resources/com/anchore/jenkins/plugins/anchore/AnchoreBuilder/help-anchoreui.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div>
+
+  Anchore Enterprise UI URL: The URL of the Anchore Enterprise. If this is set then links to the Anchore Enterprise UI will be displayed in the Anchore report seen within Jenkins.
```
Comment (juanjsebgarcia) [inline src/main/resources/com/anchore/jenkins/plugins/anchore/AnchoreBuilder/help-anchoreui.html:3]:
```suggestion
  Anchore Enterprise UI URL: The URL of the Anchore Enterprise dashboard. If this is set then links to the Anchore Enterprise UI will be displayed in the Anchore report seen within Jenkins.
```

File: src/main/webapp/js/renderOutput.js [hunk 2]
```diff
@@ -80,7 +80,7 @@ function buildPolicyEvalTable(tableId, outputFile) {
         },
         {
           "title": "Trigger ID",
-          "sTitle": "Trigger ID"
+          "sTitle": "Trigger IDe"
```
Comment (juanjsebgarcia) [inline src/main/webapp/js/renderOutput.js:83]: ?

File: src/main/webapp/js/renderOutput.js [hunk 3]
```diff
@@ -277,6 +283,21 @@ function renderLinkCell(data) {
...
+  if (typeof data == "string") {
+    data = escapeHtml(data)
+	if (URL_REGEX.test(data)) {
+		return data.replace(URL_REGEX, '<a href="$1">') + '</a>';
+	} else {
+		return data;
+	}
+  }
+  else if (typeof data == "object") {
+    return !!data
```
Comment (juanjsebgarcia) [inline src/main/webapp/js/renderOutput.js:296]: What type of `object` wouldn't be true?

```
!!{}
true


!!Object()
true 
```

File: src/main/resources/com/anchore/jenkins/plugins/anchore/AnchoreBuilder/help-anchoreui.html (hunk 1 above)
Comment (bradleyjones) [reply src/main/resources/com/anchore/jenkins/plugins/anchore/AnchoreBuilder/help-anchoreui.html:3]: 👍

File: src/main/webapp/js/renderOutput.js (hunk 2 above)
Comment (bradleyjones) [reply src/main/webapp/js/renderOutput.js:83]: no idea, must have sausage fingered will change

File: src/main/webapp/js/renderOutput.js (hunk 3 above)
Comment (bradleyjones) [reply src/main/webapp/js/renderOutput.js:296]: Good question, I was just copying the pattern below in the renderCell function, but entirely possible that has been wrong for a while. WDYT?

Comment (juanjsebgarcia) [reply src/main/webapp/js/renderOutput.js:296]: If its copying past art don't shake the boat ¯\_(ツ)_/¯

**Review by juanjsebgarcia — APPROVED** [review]
LGTM!

**Pull request merged**